hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -135,10 +138,20 @@ private void flushTransactions() {
           try(BatchOperation batchOperation = omMetadataManager.getStore()
               .initBatchOperation()) {
 
+            AtomicReference<String> lastTraceId = new AtomicReference<>();
```

Comment:
AtomicReference Do we need atomic here? As this is just used in a single thread.

Reply:
This is a workaround to update the lastTraceId from the lambda.  Java lambdas require the captured local variable must be final or effectively final.